dask_image.Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+
+FROM gpuci/miniconda-cuda:$CUDA_VER-devel-$LINUX_VER
+
+ARG CUDA_VER=11.2
+ARG PYTHON_VER=3.8
+ARG NUMPY_VER=1.20.1
+ARG RAPIDS_VER=21.08
+ARG UCX_PY_VER=0.21
```

Comment:
Good point, I _think_ this is left in so that we don't run into errors when we try to pass it as a build arg to `docker build`:

https://github.com/rapidsai/dask-build-environment/blob/1b0ceeb3bd4b28dc9273ec71d320def17de7c068/ci/gpuci/run.sh#L38

But I'm not 100% sure on how docker works with args that aren't provided. If anything, I can rewrite the build script to conditionally remove ucx-py from the build args in this case

Reply:
Interesting maybe it is safer to leave it then? WDYT Ben?

I guess there is always the possibility we add distributed tests or benchmarks to dask-image, which would make UCX useful for running those in a multi-GPU setting

Reply:
Since it isn't part of the matrix, I am okay with factoring it out in the build script - shouldn't be too much work to add it back in if we need it later 🙂